pkg/kubecost/allocation.go

```diff
@@ -535,6 +535,7 @@ type AllocationSet struct {
 	allocations  map[string]*Allocation
 	externalKeys map[string]bool
 	idleKeys     map[string]bool
+	FromSource   string
```

Comment:
It doesn't look like this made it into the codec, is that of concern?

Reply:
This has been addressed, thanks!